### DISCUSSION TITLE
what is alternative for cordova-plugin-iosrtc in capacitor

### DISCUSSION THREAD

**Discussion by pradeepmdk** [ionic-team/capacitor#3066] [General]
cordova-plugin-iosrtc doesnot work on capacitor because of hooks. so i need iosrtc for capacitor in my project

**Comment by KyDenZ** [comment]
Try run extra/hooks/iosrtc-swift-support.js with node in module root folder. Then delete the line with the hook in plugin.xml

**Comment by jcesarmobile** [comment]
if a plugin doesn't work, best thing is probably creating a Capacitor plugin

**Comment by pradeepmdk** [comment]
> Try run extra/hooks/iosrtc-swift-support.js with node in module root folder. Then delete the line with the hook in plugin.xml

after following the steps. i am getting below error.

![Screen Shot 2020-06-20 at 10 03 47 PM](https://user-images.githubusercontent.com/16595846/85206785-eede1780-b341-11ea-8169-5e3bacefe9d8.png)

**Reply by iometrine** [reply]
have you successfully installed the cordova-plugin-iosrtc  with ionic capacitor ?

**Reply by hthetiot** [reply]
@iometrine check the documentation: https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/master/docs/Building.md#capacitor
